extension.js

```diff
@@ -1,26 +1,44 @@
 const Gio = imports.gi.Gio;
 const Meta = imports.gi.Meta;
+const Mainloop = imports.mainloop;
 
 const ExtensionUtils = imports.misc.extensionUtils;
 
 const WORKSPACE_COUNT_KEY = 'workspace-count';
 const WORKSPACE_INDEX = 'workspace-index';
 const WALLPAPERS_KEY = 'workspace-wallpapers';
 const BACKGROUND_SCHEMA = 'org.gnome.desktop.background';
-const CURRENT_WALLPAPER_KEY = 'picture-uri';
+let   CURRENT_WALLPAPER_KEY = 'picture-uri';
+const INTERFACE_SCHEMA = 'org.gnome.desktop.interface';
+const SCHEMA_KEY = 'color-scheme'
 
-let _settings;
+let   _settings;
 
 function debugLog(s) {
-    //log(s);
+    log(s);
 }
 
 function _changeWallpaper() {
+	Mainloop.timeout_add(500, _changeWallpaper2 );
```

Comment:
You could use an arrow function expression instead of _changeWallpaper2. It would be cleaner than creating a new function.
It could look like this:
`Mainloop.timeout_add(500, () => {
    *code of _changeWallpaper2*
});`

Reply:
I prefer syntax without arrow. It's very orrible and not clear :)